src/components/ProductQuantity/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+
+function ProductQuantity(props) {
+  const [quantity, setQuantity] = useState(1)
+  const [price, setPrice] = useState(props.price)
+
+  const incrementQuantity = () => {
+    setQuantity(quantity + 1)
+    setPrice(price + props.price)
+  }
+
+  const decrementQuantity = () => {
+    if (quantity > 1) {
+      setQuantity(quantity - 1)
+      setPrice(price - props.price)
+    }
+  }
+
+  return (
+    <div>
+      <label htmlFor="quantity">Quantity:</label>
+      <input type="number" id="quantity" name="quantity" min="1" value={quantity} onChange={() => console.log('Quantity changed')} />
+      <button onClick={incrementQuantity}>+</button>
+      <button onClick={decrementQuantity}>-</button>
+      <p>Total Price: ${price.toFixed(2)}</p>
+    </div>
```

Comment:
Please add `type="button"` to the buttons because they are not submit buttons. 

The `console.log` is not necessary, so you can remove it.
Is there a better semantic element than the wrapping `div`?